CODEOWNERS

```diff
@@ -1,51 +1,8 @@
-# Collection Stream
-/cmd/collection/** @jordanschalm
-/engine/collection/** @jordanschalm
-/protobuf/services/collection/** @jordanschalm
+# Protocol owners are not generally differentiated by sub-area for simplicity
+/** @jordanschalm @AlexHentschel @durkmurder @zhangchiqing @peterargue @Kay-Zee
 
-# Consensus Stream
-/cmd/consensus/** @AlexHentschel @durkmurder @jordanschalm
-/engine/consensus/** @AlexHentschel @durkmurder @jordanschalm
-
-# Execution Stream
-/cmd/execution/** @zhangchiqing
-/engine/execution/** @zhangchiqing
-
-# Access Stream
-/access/** @peterargue
-/cmd/access/** @peterargue
-/cmd/observer/** @peterargue
-/engine/access/** @peterargue
-
-# Verification Stream
-/cmd/verification/** @zhangchiqing
-/engine/verification/** @zhangchiqing
-/integration/tests/verification @zhangchiqing
-
-# Ledger Stream
-/ledger/** @AlexHentschel
-
-# FVM Stream
+# FVM specific
 /fvm/** @janezpodhostnik
 
-# Networking Stream
-/network/** @Kay-Zee
-
-# Cryptography Stream
+# Cryptography 
 /crypto/** @tarakby
```

Comment:
Doesn't hurt to keep Tarak here, but the crypto package in flow-go is deprecated and replaced by https://github.com/onflow/crypto

Reply:
I removed that section and moved him into the main list. Thanks